packages/server/graphql/mutations/setTaskEstimate.ts

```diff
@@ -307,6 +307,7 @@ const setTaskEstimate = {
         break
       }
       default:
+        success = true
```

Comment:
-1 Works as expected, but I would prefer if this switch fails once we add another service without fixing it here.
I propose changing the key to `undefined` and skip the `default` one altogether.
```ts
  undefined:
    success = true
    break
}
```